packages/discovery-provider/src/api/v1/users.py

```diff
@@ -3201,6 +3201,8 @@ def get(self, receiving_user_id, grantor_user_id):
     {
         "id": fields.String(required=True),
         "user_id": fields.String(required=False),
+        "entity_id": fields.String(required=True),
+        "entity_type": fields.String(required=True),
```

Comment:
Do you mind splitting the discovery/sdk changes into a separate PR and merging that first?
That way we can make sure it's on all nodes before shipping the SDK change that consumes the new fields.

Reply:
Also wondering if we should use an `enum` for this similar to [here](https://github.com/AudiusProject/audius-protocol/blob/1c61e0b461d0c3fe71465c6abbfa94e7a12c3088/packages/discovery-provider/src/api/v1/models/activities.py#L39)

That way, the generated SDK types will show only those literals instead of a generic `string`.

We may also want to write an `extend_` helper for our response model and use it to lowercase the type when we serialize this model so it's `'track' | 'comment'` since we usually lowercase values like this.